vividus-tests/src/main/resources/properties/environment/system/environment.properties

```diff
@@ -0,0 +1,14 @@
+selenium.grid.enabled=true
+selenium.grid.url=https://${BROWSERSTACK_USER}:${BROWSERSTACK_KEY}@hub-cloud.browserstack.com/wd/hub
+
+selenium.grid.capabilities.browser=Chrome
+selenium.grid.capabilities.os=Windows
+selenium.grid.capabilities.os_version=10
+selenium.grid.capabilities.resolution=1920x1080
+selenium.grid.capabilities.build=Vividus System Tests
```

Comment:
also please add 
```properties
selenium.grid.capabilities.project=Vividus
```
to group builds

Reply:
also what about W3C protocol: https://www.browserstack.com/automate/capabilities?tag=selenium-4 ?